lox_services/persistence/storage/constants.py

```diff
@@ -1 +1,7 @@
 INTERNAL_COMPANIES = ["Test", "test", "Lox", "lox"]
+
+ORIGINAL_INVOICES_BUCKET = "invoices_email"
+
+CLIENT_INVOICES_BUCKET = "invoices_client"
```

Comment:
there is an S at the end of client: `invoices_clients`

Reply:
cf[ #317](https://github.com/Lox-Solution/Main-Algorithms/pull/317) from Main aglo